anyway/models.py

```diff
@@ -174,6 +174,7 @@ class Marker(MarkerMixin, Base): # TODO rename to AccidentMarker
     cross_mode = Column(Integer)
     cross_location = Column(Integer)
     cross_direction = Column(Integer)
+    involved = relationship("Involved", foreign_keys="Involved.accident_id")
```

Comment:
Does this require a migration?

Reply:
No, as it does not effect the database itself. It's an internal SQLAlchemy mapping.